system_tests/espresso_finality_node_test.go

```diff
@@ -0,0 +1,86 @@
+package arbtest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+)
+
+func createL1AndL2NodeEspressoFinalityNode(t *testing.T, builder *NodeBuilder) (*TestClient, func()) {
```

Comment:
Let's separate the L1L2 node and the finality node. I think this is the common case of how people use finality node

Reply:
Sorry the naming is confusing but its only creating the finality node!

Updated the naming!